def raise_exception_msg(msg):
    raise NameError(msg)
